grammar.js

```diff
@@ -319,6 +320,35 @@ module.exports = grammar({
       $._expression
     ),
 
+    binary_operator: $ => choice(
+      caseInsensitive('-eq'),
+      caseInsensitive('-ne'),
+      caseInsensitive('-ge'),
+      caseInsensitive('-gt'),
+      caseInsensitive('-lt'),
+      caseInsensitive('-le'),
+      caseInsensitive('-like'),
+      caseInsensitive('-notlike'),
+      caseInsensitive('-match'),
+      caseInsensitive('-notmatch'),
+      caseInsensitive('-replace'),
+      caseInsensitive('-contains'),
+      caseInsensitive('-notcontains'),
+      caseInsensitive('-in'),
+      caseInsensitive('-notin'),
+      caseInsensitive('-split'),
+      caseInsensitive('-join'),
+      caseInsensitive('-is'),
+      caseInsensitive('-isnot'),
+      caseInsensitive('-as'),
+    ),
+
+    binary_operator_statement: $ => seq(
+      $.variable,
```

Comment:
I would have expected this to also allow an expression on the LHS. I see the tests only look variable binop comparisons, rather than something like ``(gci | % { $_.filepath }) -join "`n"``